app/helpers.py

```diff
@@ -173,4 +175,49 @@ def gen_aq_score(db_conn, city):
         ret_val["score"] = 2
     elif combined_aq <= buckets[5]:
         ret_val["score"] = 1
-    return ret_val
\ No newline at end of file
+    return ret_val
+
+def gen_city_weight(crime=0, walk=0, air=0, rent=0):
+  """
+  gen_city_weight returns a dict of float city score weights
+  that sum to 1.0 (given passed integer weight values: 0-10)
+
+  example return value:
+    {"crime": 0.4, "walk": 0.3, "air": 0.1, "rent": 0.2}
+  """
+  # Validate inbound parameters (do parameter values sum to 10)
+  wght_sum = crime + walk + air + rent
+  if wght_sum != 10:
```

Comment:
I need to clarify how the user sets the weights. To me it doesn't seem like they would always sum to 10?

Reply:
@PalmerTurley34 I think I need to circle back with Chad and see how exactly they are going to code the dimension sliders.  My running assumptions are:

- the user can pick a slider value of 1 to 10
- all of the slider values must sum to 10 and that will be enforced by the front end

If you're cool with the rest of this PR, my suggestion would be to merge the code and I/we can circle back with Chad and close on the frontend assumptions (and I can update the DS backend if they opt for something else).  This way we've got a non-mock handler that the frontend guys can play with